src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+body {
```

Comment:
The 'body' selector should not be in the 'card.scss' file. It's a global style and it should be declared in a general styles file.